test/content/ContentTests.js

```diff
@@ -159,89 +158,89 @@ describe('Content Contract Tests', () => {
 
     describe("Burn", () => {
         it('Burn Assets', async () => {
-            const signature = await sign(playerAddress.address, [1, 2], [10, 75], 1, craftingSystemAddress.address, content.address);
-            var mintData = [playerAddress.address, [1, 2], [10, 75], 1, craftingSystemAddress.address, signature];
+            const signature = await sign(playerAddress.address, [0, 1], [10, 75], 1, craftingSystemAddress.address, content.address);
+            var mintData = [playerAddress.address, [0, 1], [10, 75], 1, craftingSystemAddress.address, signature];
             await content.connect(playerAddress).mintBatch(mintData);
     
-            var burnData = [playerAddress.address, [1, 2], [5, 25]];
+            var burnData = [playerAddress.address, [0, 1], [5, 25]];
             expect (await content.connect(playerAddress).burnBatch(burnData))
                 .to.emit(content, "Burn");
 
-            expect(await content.connect(playerAddress).totalSupply(1)).to.equal(5);
-            expect(await content.connect(playerAddress).totalSupply(2)).to.equal(50);
+            expect(await content.connect(playerAddress).totalSupply(0)).to.equal(5);
+            expect(await content.connect(playerAddress).totalSupply(1)).to.equal(50);
     
             await content.connect(playerAddress).setApprovalForAll(craftingSystemAddress.address, true);
             await content.connect(craftingSystemAddress).burnBatch(burnData);
-            expect(await content.connect(playerAddress).totalSupply(1)).to.equal(0);
-            expect(await content.connect(playerAddress).totalSupply(2)).to.equal(25);
+            expect(await content.connect(playerAddress).totalSupply(0)).to.equal(0);
+            expect(await content.connect(playerAddress).totalSupply(1)).to.equal(25);
             
-            expect(await content.balanceOf(playerAddress.address, 1)).to.equal(0);
-            expect(await content.balanceOf(playerAddress.address, 2)).to.equal(25);
+            expect(await content.balanceOf(playerAddress.address, 0)).to.equal(0);
+            expect(await content.balanceOf(playerAddress.address, 1)).to.equal(25);
         });
         
         it('Invalid burns', async () => {
-            const signature = await sign(playerAddress.address, [1], [10], 1, craftingSystemAddress.address, content.address);
-            var mintData = [playerAddress.address, [1], [10], 1, craftingSystemAddress.address, signature];
+            const signature = await sign(playerAddress.address, [0], [10], 1, craftingSystemAddress.address, content.address);
+            var mintData = [playerAddress.address, [0], [10], 1, craftingSystemAddress.address, signature];
             await content.connect(playerAddress).mintBatch(mintData);
     
-            var burnData = [playerAddress.address, [1], [5]];
+            var burnData = [playerAddress.address, [0], [5]];
 
             await expect(content.connect(lootboxSystemAddress).mintBatch(burnData)).to.be.reverted;
```

Comment:
My mistake, these 2 lines should be burnBatch functions not mintBatch functions.